cli/internal/cache/cache.go

```diff
@@ -53,39 +55,42 @@ type cacheMultiplexer struct {
 }
 
 func (mplex cacheMultiplexer) Put(target string, key string, duration int, files []string) error {
-	mplex.storeUntil(target, key, duration, files, len(mplex.caches))
-	return nil
+	return mplex.storeUntil(target, key, duration, files, len(mplex.caches))
 }
 
 // storeUntil stores artifacts into higher priority caches than the given one.
 // Used after artifact retrieval to ensure we have them in eg. the directory cache after
 // downloading from the RPC cache.
 // This is a little inefficient since we could write the file to plz-out then copy it to the dir cache,
 // but it's hard to fix that without breaking the cache abstraction.
-func (mplex cacheMultiplexer) storeUntil(target string, key string, duration int, outputGlobs []string, stopAt int) {
+func (mplex cacheMultiplexer) storeUntil(target string, key string, duration int, outputGlobs []string, stopAt int) error {
 	// Attempt to store on all caches simultaneously.
-	var wg sync.WaitGroup
+	g := new(errgroup.Group)
```

Comment:
Prefer `g := &errgroup.Group{}` to `new()` usages, based on [style guide](https://github.com/uber-go/guide/blob/master/style.md#initializing-struct-references)

Reply:
Done.